graphql/database/experiments/features.js

```diff
@@ -198,6 +199,47 @@ const features = {
       },
     ],
   },
+  [SFAC_EXTENSION_PROMPT]: {
+    defaultValue: 'Control',
```

Comment:
Though this won't yet be used, users would be bucketed into the experiment if we deploy this PR: calling `user.features` in schema will get all features and bucket the user. This would mess with analytics.

Does that sound right to you? If so, comment out the actual experiment here until we're ready to launch.

Reply:
I thought that it wouldn't if there's no coverage yet. No one does get bucketed in that case no?

Reply:
@jedtan That's my bad, I missed the coverage property 🤦 This should be all good.